src/pages/Dashboard/Event.tsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+// import ReactDOM from 'react-dom/client'
+import { useState, useEffect } from 'react';
+import { useParams, useNavigate } from 'react-router-dom';
+import { eventType } from '../../types/api';
+import "./Event.css"
+import "./Dashboard.css"
+import logo from '/assets/pmc_logo.svg';
+
+
+const Event: React.FC = () => {
+    const [event, setEvent] = useState<eventType | null>(null);
+    const { event_id } = useParams<{ event_id: string }>();
+    const [loading, setLoading] = useState(true);
+    const [memberId, setMemberId] = useState<string | undefined>(undefined);
+    const [displayName, setDisplayName] = useState<string | undefined>(undefined);
+    const [isLoggedIn, setIsLoggedIn] = useState(false);
+    const navigateTo = useNavigate();
+
+    function checkMember() {
+        try {
+            const loggedIn = localStorage.getItem('member_id') !== null;
+            const id = localStorage.getItem('member_id') ?? undefined;
+            const displayName = localStorage.getItem('member_name') ?? "guest";
```

Comment:
You can use the auth object from firebase.ts to access the currently signed-in user.
```
// auth from firebase.ts
auth.currentUser
```
returns the currently signed in user or null. The user object returned contains uid, email, displayName, etc.